### DISCUSSION TITLE
how to get  one key's value from a json

### DISCUSSION THREAD

**Discussion by huangmingyou** [lowdefy/lowdefy#881] [Questions and Help] [answered]
http api return a json like 
```json
{
"key1":"value1",
"key2":"value2"
}
```
I want get the key2's value ,how to do this?

**Comment by SamTolmay** [comment, marked as answer]
Hi @huangmingyou 

If you are using the `AxiosHttp` requests, the response probably looks like(in YAML):
```yaml
status: 200
data:
  key1: value1
  key2: value2
headers:
  # ...
```

To get the value in `key2`, you can use the [`_request`](https://docs.lowdefy.com/_request) operator. The syntax is:
```yaml
_request: my_request_id.data.key2
```

All "getter" operators like `_state`, `_global` or `_url_query` work with the same syntax.


You can also have a look at the [`_get`](https://docs.lowdefy.com/_get) operator. This is used if you want to specify the item that you want to get the value from.

**Reply by huangmingyou** [reply]
thank you!
